test/mail/parsers/rfc_2822_test.exs

```diff
@@ -236,18 +236,34 @@ defmodule Mail.Parsers.RFC2822Test do
     assert message.headers["x-reallylongheadernamethatcausesbodytowrap"] == "BodyOnNewLine"
   end
 
-  test "parses headers even when does not follow usual format: Upper-Dash-Upper" do
+  test "parses a message even when headers do not follow usual format: Upper-Dash-Upper" do
     message = parse_email("""
     X-all-lower-case: ImportantValue
+    To: user@example.com
     Cc: "User, First" <first@example.com>, "User, Second" <second@example.com>, third@example.com
+    From: me@example.com
+    Reply-to: otherme@example.com
+    Subject: Test Email
+    Message-id: <SELF@MESSAGE.example.test.domain.com>
```

Comment:
@KiyoMenager and everybody.

I suggest to not use email addresses or domains, which we does not own, or which have been officially reserved for testing purposes.
Even when this email is not to be sent to those addressess.
Let's just don't do that.

Reply:
We can do that within gustav @slavo2, but in this case I have to follow the overall lib style.
Also this is an email parser/renderer lib, they do not mean to send any emails.

Reply:
Well, I would still suggest to use at least some official example domains (such as example.com  and example.org ) and possibly its subdomains, rather than existing domain.com

Reply:
Oh okok, I better understand what you meant now (I was looking at this line `Reply-to: otherme@example.com`).

Will change that / be careful next time.

Reply:
Huh, interesting. I didn't know about the offical status of example.com/org. Thanks @slavo2!

Reply:
You are welcome. :)

There are even phone number ranges, which are dedicated for fictional usage, I prefer to use them in my seed / tests as well. 
For US 555 - 01xx range
https://en.wikipedia.org/wiki/Fictitious_telephone_number#North_American_Numbering_Plan
https://en.wikipedia.org/wiki/Fictitious_telephone_number